Makefile

```diff
@@ -119,7 +119,7 @@ build_container:
        -f $(DOCKERFILE) \
 		.
 
-container: sonobuoy
+containers: build/linux/arm64/sonobuoy build/linux/amd64/sonobuoy
```

Comment:
I can't leave a comment on all the lines that need to be updated, but the following targets need to have their dependencies updated to use `containers`:
* `all`
* `deploy_kind`
* `push`